src/app/navbar/navbar.component.ts

```diff
@@ -38,8 +39,8 @@ export class NavbarComponent extends Logout implements OnInit {
   isExtended = false;
   Dockstore = Dockstore;
   protected ngUnsubscribe: Subject<{}> = new Subject();
-  private currentTOSVersion: User.TosversionEnum = User.TosversionEnum.TOSVERSION1;
-  private currentPrivacyPolicyVersion: User.PrivacyPolicyVersionEnum = User.PrivacyPolicyVersionEnum.PRIVACYPOLICYVERSION25;
+  private currentTOSVersion: User.TosversionEnum = currentTOSVersion;
```

Comment:
I would make this and the next one `readonly` (unless they can change, which I don't think they do, but just starting to look at PR).